fe/fe-core/src/main/java/com/starrocks/clone/DiskAndTabletLoadReBalancer.java

```diff
@@ -1646,6 +1655,10 @@ private Map<Pair<Long, Long>, PartitionStat> getPartitionStats(TStorageMedium me
                         continue;
                     }
 
+                    if (!olapTbl.needSchedule(isLocalBalance)) {
```

Comment:
This judgment has already been made above.

Reply:
// check table is olap table or colocate table
                    if (!table.needSchedule(isLocalBalance)) {
                        continue;
                    }